src/main/java/com/exasol/adapter/document/documentfetcher/files/csv/CsvIterator.java

```diff
@@ -2,55 +2,58 @@
 
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.util.List;
 import java.util.NoSuchElementException;
 
 import com.exasol.adapter.document.documentfetcher.files.InputDataException;
 import com.exasol.adapter.document.documentfetcher.files.RemoteFile;
 import com.exasol.adapter.document.documentnode.DocumentNode;
 import com.exasol.adapter.document.documentnode.csv.CsvObjectNode;
 import com.exasol.adapter.document.iterators.CloseableIterator;
+import com.exasol.adapter.document.mapping.ColumnMapping;
 import com.exasol.errorreporting.ExaError;
 
-import de.siegmar.fastcsv.reader.CsvReader;
-import de.siegmar.fastcsv.reader.CsvRow;
-import de.siegmar.fastcsv.reader.NamedCsvReader;
-import de.siegmar.fastcsv.reader.NamedCsvRow;
+import de.siegmar.fastcsv.reader.*;
 
 /**
- * This class iterates the lines of a JSON-Lines file an creates for each line a JSON {@link DocumentNode}.
+ * This class iterates the lines of a CSV file and creates a {@link CsvObjectNode} for each line.
  */
 class CsvIterator implements CloseableIterator<DocumentNode> {
     // efficient reading of characters, arrays and lines
     private final InputStreamReader inputStreamReader;
-    private long lineCounter = 0;
+    private final List<ColumnMapping> csvColumns;
```

Comment:
Is the field `csvColumns` unused by intention?

Reply:
Yes. It will be used soon.